sagemcom_api/action_error_exception_handler.py

```diff
@@ -0,0 +1,75 @@
+"""Logic to spot and create ActionErrorExceptions"""
+
+from sagemcom_api.const import XMO_ACCESS_RESTRICTION_ERR, XMO_AUTHENTICATION_ERR, XMO_LOGIN_RETRY_ERR, XMO_MAX_SESSION_COUNT_ERR, XMO_NO_ERR, XMO_NON_WRITABLE_PARAMETER_ERR, XMO_REQUEST_ACTION_ERR, XMO_UNKNOWN_PATH_ERR
+from sagemcom_api.exceptions import AccessRestrictionException, AuthenticationException, LoginRetryErrorException, MaximumSessionCountException, NonWritableParameterException, UnknownException, UnknownPathException
+
+
+class ActionErrorHandler:
+    """Raised when a requested action has an error"""
+
+    KNOWN_EXCEPTIONS = (
+        XMO_AUTHENTICATION_ERR,
+        XMO_ACCESS_RESTRICTION_ERR,
+        XMO_NON_WRITABLE_PARAMETER_ERR,
+        XMO_UNKNOWN_PATH_ERR,
+        XMO_MAX_SESSION_COUNT_ERR,
+        XMO_LOGIN_RETRY_ERR
+    )
+
+    @staticmethod
+    def ThrowIf(response):
```

Comment:
Could you adhere to the PEP8 styleguide? https://peps.python.org/pep-0008/#function-and-variable-names. In Python function names should be lower case with words separated by underscores.

Reply:
And good to run `pre-commit` to see if the linter is happy :-).

Reply:
No worries. Still plan to move the formatting and linting to ruff and the project to uv, but will need some time first 😃